cookbook/lists/long-lists.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: "Working with long lists"
```

Comment:
The xample here doesn't use a long list; maybe we can use a generator or some such to demonstrate a practical example with an infinite number of items?

For inspiration, check out the new getting started tutorial: https://github.com/filiph/startup_namer_flutter

Reply:
Good feedback. I'll fix that up tomorrow morning!